.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.6.9
```

Comment:
Was there a blocker to not use Ruby 3 ?

Reply:
It just makes more sense better to develop with the lowest supported version. If you code using Ruby 3 and use Ruby 3 features, your code will only fail in CI.